app-directory/css-in-js/README.md

```diff
@@ -0,0 +1,41 @@
+# css-in-js example
+
+This example shows how use CSS-in-JS libraries within app directory
+
+## Demo
+
+https://app-dir-css-in-js.vercel.app
+
+## How to Use
+
+You can choose from one of the following two methods to use this repository:
+
+### One-Click Deploy
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=vercel-examples):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/app-directory/css-in-js&project-name=css-in-js&repository-name=css-in-js)
+
+### Clone and Deploy
+
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npx create-next-app --example https://github.com/vercel/examples/tree/main/app-directory/css-in-js
+# or
+yarn create next-app --example https://github.com/vercel/examples/tree/main/app-directory/css-in-js
+```
+
+Next, run Next.js in development mode:
+
+```bash
+npm install
+npm run dev
+
+# or
+
+yarn
+yarn dev
+```
+
+Deploy it to the cloud with [Vercel](https://vercel.com/new?utm_source=github&utm_medium=readme&utm_campaign=edge-middleware-eap) ([Documentation](https://nextjs.org/docs/deployment)).
```

Comment:
These instructions should be pnpm only if the example has a pnpm-lock, otherwise none of these commands will work.

Reply:
`pnpm-lock` is added to monorepos .gitignore, should we remove this then?